db/controllers/articlesbyid.controller.js

```diff
@@ -0,0 +1,20 @@
+const { fetchArticleById } = require("../../db/models/articlebyid.model");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  if (!article_id || isNaN(Number(article_id))) {
```

Comment:
What does psql do when you try to pass it an invalid article id? Could we take advantage of this instead of trying to manually check this ourselves? As a rule of thumb, always try making the request and if psql gives you an error to work with, use it - and if it doesn't but you expect to send an error then that's when you need a custom one!